data/remnant ships.txt

```diff
@@ -348,6 +347,87 @@ ship "Gull" "Gull (Bunks)"
 
 
 
+ship "Ibis"
+	sprite "ship/ibis"
+	thumbnail "thumbnail/ibis"
+	attributes
+		category "Medium Warship"
+		licenses
+			"Remnant Capital"
+		"cost" 30000000
+		"shields" 22000
+		"hull" 11000
+		"required crew" 20
+		"bunks" 48
+		"mass" 300
+		"drag" 7.1
+		"heat dissipation" 0.7
+		"fuel capacity" 700
+		"ramscoop" 2
+		"outfit space" 568
+		"cargo space" 103
+		"weapon capacity" 200
```

Comment:
*Exactly* 200? As a medium warship and a carrier, I'd expect less weapon space (and maybe more engine space instead of the exact same as the Albatross).

Reply:
Weapon Space reduced to 180, and engine space increased to 160. How's that look?

Reply:
As with above, factions outside Republic space don't tend to have such "nice" numbers. Fudge them up or down a bit.

Reply:
OK. Made them not nice.